api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/queries/GetMotTestCurrentByVehicleId.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.dvsa.mot.persist.jdbc.queries;
+
+@SuppressWarnings({"checkstyle:OperatorWrap", "checkstyle:Indentation"})
+public class GetMotTestCurrentByVehicleId {
+
+    public String buildQuery(boolean includeAppleals) {
```

Comment:
I like the moving out of smaller components of the SQL queries from MotTestReadSql into separate single responsibility Classes.  Maybe something to consider doing across the board :)